### PR TITLE
Added per employee query for survey

### PR DESCRIPTION
**Pull request by mjennings061**
Added per_employee to application.py and stats_helper

### CODE REVIEW

File: server/application.py [hunk 1]
```diff
@@ -24,6 +24,11 @@ def alldata():
     return json.dumps(stats_helper.join_all())
```
Comment (petergmurphy) [inline server/application.py:25]: There should only be one blank line between methods inside of a class according to PEP8

**Review by petergmurphy — CHANGES_REQUESTED** [review]
There should only be one blank line between methods inside of a class according to PEP8

File: server/application.py (hunk 1 above)
Comment (mjennings061) [reply server/application.py:25]: Removed blank lines

**Comment by mjennings061** [comment]
PEP8 changes made

**Review by petergmurphy — APPROVED** [review]
Approved

**Pull request merged**